doc/JSON_INFO.md

```diff
@@ -216,10 +216,10 @@ The syntax listed here is still valid.
   [ "fur", 1 ]
 ],
 [
-  [ "plant_fibre", 20 ],
-  [ "sinew", 20 ],
-  [ "thread", 20 ],
-  [ "duct_tape", 20 ]
+  [ "plant_fibre", 20, false ], // Optional flag for recoverability, default is true.
```

Comment:
Yeah, I saw the falses in the archery PR and thought they were related to this.  But that's because I saw this first.